framework/wazuh/security.py

```diff
@@ -1317,6 +1317,7 @@ async def get_security_config() -> WazuhResult:
     return WazuhResult({'data': get_security_conf()})
 
 
+# TODO(26553) - To be removed
```

Comment:
Users will still be able to update the security configuration, we should update its logic to do so here

Reply:
Fixed in 295ed002d916bd363c2b9284c18e97d79ebe89ed.